composer.json

```diff
@@ -12,11 +12,11 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/symfony": "~2.3",
-        "symfony/phpunit-bridge": "~2.7@dev",
+        "symfony/phpunit-bridge": "~2.7",
         "symfony/monolog-bundle": "~2.1",
         "doctrine/common": "~2.4",
         "doctrine/doctrine-bundle": "~1.0",
-        "doctrine/phpcr-odm": "~1.1,>=1.1.0-RC1",
+        "doctrine/phpcr-odm": "~1.3",
```

Comment:
I would indeed only bump it where necessary, not many bundles need a higher dep.